lesson_02/job1/dal/local_disk.py

```diff
@@ -1,6 +1,19 @@
 from typing import List, Dict, Any
+import os
+import json
+
+
+BASE_DIR = os.environ.get("BASE_DIR")
 
 
 def save_to_disk(json_content: List[Dict[str, Any]], path: str) -> None:
-    # TODO: implement me
-    pass
+    date = json_content[0]
```

Comment:
What if `json_content[0]` becomes not a date one day?
I would:
- validate this to be sure it's a date
- pass the date as a parameter
The latter is preffered